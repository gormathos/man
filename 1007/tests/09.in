-24;5;1;97;20;-78;-100;-50;4;22
-96;5;1;9;-66;35;15;-63;4;19
13;5;1;-37;99;-71;-92;50;4;18
60;5;1;-72;-34;97;-16;-23;4;-94
-30;5;1;-34;34;-85;-37;-95;4;52
-17;5;1;83;26;-30;-85;-36;4;-88
30;5;1;27;-75;-25;71;-94;4;-42
40;5;1;-43;-91;-55;-17;70;4;26
77;5;1;3;-66;-10;-42;85;4;14
-46;5;1;-51;-92;-77;37;-84;4;55
