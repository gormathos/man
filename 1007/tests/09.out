-24;97;20;-78;-100;-50;22
-96;9;-66;35;15;-63;19
13;-37;99;-71;-92;50;18
60;-72;-34;97;-16;-23;-94
-30;-34;34;-85;-37;-95;52
-17;83;26;-30;-85;-36;-88
30;27;-75;-25;71;-94;-42
40;-43;-91;-55;-17;70;26
77;3;-66;-10;-42;85;14
-46;-51;-92;-77;37;-84;55
